src/actors/symbolication.rs

```diff
@@ -245,10 +246,10 @@ impl SymbolicationActor {
         }
     }
 
-    fn create_symbolication_request<F, R>(&self, f: F) -> RequestId
+    fn create_symbolication_request<F>(&self, f: F) -> RequestId
     where
-        F: FnOnce() -> R,
-        R: Future<Item = CompletedSymbolicationResponse, Error = SymbolicationError> + 'static,
+        F: std::future::Future<Output = Result<CompletedSymbolicationResponse, SymbolicationError>>
```

Comment:
Can you import this as `Future` and import the legacy future as `Future01` or `Future1`?